### ISSUE TITLE
Not showing target humidity for LEVOIT Superior 6000S

### ISSUE THREAD

**Issue by bloggermanic** [pschroeder89/homebridge-levoit-humidifiers#71]
**Describe The Bug:**

Using the plugin, the humidity setting shows 0 when it is current set to a higher value. 
Using LEVOIT Superior 6000S. 

**To Reproduce:**

When setting the target humidity value on the manufacturers app to 43. The HomeKit app say rising to 0% and this never changes. 

**Expected behavior:**

I expect the HomeKit app to reflect the target humidity. 

**Logs:**

```
Show the Homebridge logs here, remove any sensitive information.
```

**Plugin Config:**

```json
Show your Homebridge config.json here, remove any sensitive information.
```

**Screenshots:**

![image](https://github.com/pschroeder89/homebridge-levoit-humidifiers/assets/74580384/10548653-af99-4cba-b607-919a342b93ce)
**Environment:**

* **Plugin Version**:
* **Homebridge Version**: 
Hoobs core 5.0.12
* **Node.js Version**: 
* **NPM Version**: 
* **Operating System**: 
hoobs

**Comment by pschroeder89** [comment]
I'm gonna need some debug logs from the plugin since I don't have this device. Turn debugging on, then change all of the settings one by one (set target humidity, turn on / off sleep mode, etc). Post the logs here. Also, can you share the plugin version you're using? I made some recent changes around this model and just want to make sure you're up to date. Thanks!

**Comment by pschroeder89** [comment]
Oh, you have auto mode on. I was told this model doesn't support changing humidity levels when Auto is turned on, so we return 0. Is that incorrect?

**Comment by bloggermanic** [comment]
Oh. Yes. It is in auto mode. I guess that's why u return a 0. I'll see if the problem exists in manual mode

**Comment by pschroeder89** [comment]
I'm messing with this in 1.13.0-dev6 if you want to test it out. Even in auto mode, it will display the correct humidity level.

The reason why I set it to 0 initially was because I thought it would prevent people from trying to set the humidity level while in auto or sleep modes. But I'll make an update to the README that doing that will take the user out of those modes. I like seeing the correct humidity % instead of the 0, so I think I'll go forward with this change if I can get some other people to test it out.

**Comment by pschroeder89** [comment]
This is now published in 1.14.1
